src/tests/Microsoft.PowerFx.Core.Tests/ExpressionTestCases/Index.txt

```diff
@@ -0,0 +1,41 @@
+>> Index([10, 20, 30, 40, 50], 3)
+{Value:30}
+
+>> Index([1, 2, 3, 4, 5], 6)
+#Error
+
+>> Index([1, 2, 3, 4, 5], -1)
+#Error
+
+>> Index([1, 2, 3, 4, 5], Blank())
+Blank()
+
+>> Index(Blank(), 2)
+Blank()
+
+>> Index(Sort([1,2,7,3,4,2],1/Value), 1)
+{Value:7}
+
+>> Index([1, 2, 3, 4, 5], 1/0)
```

Comment:
Other cases:

- 1st argument is an error, returns error.

Reply:
added